src/slider/__tests__/slider.test.js

```diff
@@ -0,0 +1,186 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import React from 'react';
+import {mount} from 'enzyme';
+import {Slider, StyledThumb} from '../index';
+
+/* global document */
+/* global window */
+
+const originalDocumentAddListener = document.addEventListener;
+
+describe('Stateless slider', function() {
+  let wrapper;
+  let allProps: {
+      range: Array<number>,
+      value: Array<number>,
+      step: ?number,
+      onChange: () => void,
+    },
+    mockFn;
+
+  beforeEach(function() {
+    //$FlowFixMe
+    document.addEventListener = jest.fn();
```

Comment:
We clear mock after each test therefore we can do this assignment only once in the `beforeAll`

Reply:
ok